.github/workflows/release.yml

```
@@ -0,0 +1,99 @@
+name: "release"
+
+on:
+  push:
+    tags:
+      - "*"
```

```suggestion
      - "v*"
```

I've just added a tag rule to prevent general creation of tags with this pattern. I'm assuming this won't be blocked during the "Create Release" flow, but we can change that if needed when we cut a release.